dist/index.html

```diff
@@ -20,30 +26,34 @@
         <nav class="main-header__nav">
           <div class="nav__navigation left">
             <span class="logo">
-              FITNESS
+              <a href="#">FITNE<span class="orange">SS</span></a>
             </span>
-            <ul class="nav-links">
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
-              <li>link</li>
+            <ul class="nav-links hide">
```

Comment:
the `hide` class is redundant here, doesn't have corresponding styles

Reply:
removed